src/components/ResultsView.tsx

```diff
@@ -122,15 +122,17 @@ const ResultHeader = ({
   );
 };
 
+
 const CellEmbed = ({ uid }: { uid: string }) => {
+  const isPage = !!getPageTitleByPageUid(uid) ? "page-embed" : "block-embed"
   const contentRef = useRef(null);
   useEffect(() => {
     window.roamAlphaAPI.ui.components.renderBlock({
       uid,
       el: contentRef.current,
     });
   }, [contentRef]);
-  return <div ref={contentRef} className="roamjs-query-embed" />;
+  return <div ref={contentRef} className={`roamjs-query-embed ${isPage}`} />
```

Comment:
nice - let's also display the title of the page somewhere in this markup. a simple `<h1 className="rm-page-title">` within the `div` when `isPage` is true should be sufficient

Reply:
I'm having trouble getting the `h1` to stay.  It seems `renderBlock` keeps nuking it.
I've tried adding it inside the `useEffect` adding a separate `useEffect` as well as in the `return`

Would this require a larger refactor or am I missing something simple?

Reply:
or we can nest the `ref={contentRef}` ... one sec ...

Reply:
yup this is how we want to do this - it's fine if the title is not editable.

Let's also add a margin-left to the title and remove the margin-top/bottom